pkg/vc/issuer/credentials.go

```diff
@@ -28,22 +28,23 @@ const (
 	// DIDConnectCredentialType vc type.
 	DIDConnectCredentialType = "DIDConnection"
 
-	// DIDCommInitCredentialType vc type.
-	DIDCommInitCredentialType = "DIDCommInit"
+	// ConsentCredentialType vc type.
+	ConsentCredentialType = "ConsentCredential"
```

Comment:
https://github.com/trustbloc/edge-adapter/blob/d8da8942f2763447620d8203381fb86284f27bd6/pkg/vc/models.go#L13-L12

Reply:
Here is the json-ld for Consent Credential - https://trustbloc.github.io/context/vc/consent-credential-v1.jsonld. I believe, refactoring of Consent Credential in RP Adapter is part of https://github.com/trustbloc/edge-adapter/issues/149.

Reply:
I have [moved](https://github.com/trustbloc/edge-adapter/compare/f896e797052280e7990e4cd01b2ea157c2e38190..a34899ab48491414dfe041f88128d2c1cdead8cc#diff-4452039e037010c161e6e236076164d9R11) the CC and related models to common package as it would be used by both the RP and the Issuer.

Reply:
This doesn't follow convention. See for instance `UniversityDegreeCredential`.

Reply:
I'm confused. It's a `Consent` and named it as `ConsentCredential`. What do you want me to update ?

Reply:
now i'm confused as well

nevermind

Reply:
Also, its the name used in the jsonld context - https://trustbloc.github.io/context/vc/consent-credential-v1.jsonld.

Reply:
@rolsonquadras just for my curiosity: why was it changed?

Reply:
json-ld context was defined yesterday - https://github.com/trustbloc/context/pull/14/files. I didn't even remember we had some constant in our code to define the ConsentCredential.